master/master.cfg

```diff
@@ -126,13 +126,15 @@ class FreezeBuild(TaggedBuildFactory):
 
 class UnixBuild(TaggedBuildFactory):
     configureFlags = ["--with-pydebug"]
+    configureEnv = {}
```

Comment:
It seems like an empty dict is fine, since it's only *added* environment variables, it doesn't *replace* the full environment as env= of subprocess.Popen.